.dockerignore

```diff
@@ -20,3 +20,6 @@ tf-docs-crawled/
 truefoundry-docs/
 *.zip
 ragenv/
+volumes/
+qdrant_db/
```

Comment:
Arent qdrant ones part of volumes/ folder which is already ignore

Reply:
This one was when we start a separate qdrant container. Just added that to ignore.